test/jdk/tools/launcher/modules/basic/LauncherErrors.java

```diff
@@ -92,9 +92,10 @@ public void testErrorMessage() throws Exception {
         ProcessTools.executeTestJava("-Djava.security.manager", "--module-path", dir, "--module", mid)
                     .outputTo(System.out)
                     .errorTo(System.out)
-                    .shouldContain("Error: Unable to initialize main class " + MAIN_CLASS + " in module " + TEST_MODULE)
-                    .shouldContain("Caused by: java.security.AccessControlException: access denied")
+                    .shouldContain("Error: Unable to load main class " + MAIN_CLASS + " in module " + TEST_MODULE)
+                    .shouldContain("java.security.AccessControlException")
                     .shouldNotHaveExitValue(0);
```

Comment:
The updates to the launcher tests look okay but I'm wondering if the "Caused by:" is the output now? I get it that the "Unable to ..." message has changed but has the format of the cause changed too?

I assume you'll fix up the copyright date on Arrrrgs and LauncherErrors before integrating.

Reply:
"Caused by:" has (always) been missing from the named module version of the error message. I'll add in and modify the test accordingly. Updated copyright.